packages/cosmos_ui_components/lib/cosmos_theme.dart

```diff
@@ -84,9 +87,14 @@ class CosmosThemeData extends Equatable {
     this.elevationL = defaultElevationL,
     this.colors = const CosmosColorsData(),
     this.brightness = Brightness.light,
-    this.errorColor = CosmosColorsData.defaultError,
   });
 
+  const CosmosThemeData.dark() : this(colors: darkThemeColors, brightness: Brightness.dark);
+
+  const CosmosThemeData.light() : this(colors: lightThemeColors, brightness: Brightness.light);
+
+  ThemeData buildFlutterTheme() => convertCosmosThemeToMaterialTheme(this);
```

Comment:
Added this function as per our discussion. Do let me know if this is okay. I couldn't figure out a way to pass the CosmosTheme to this converter.

Reply:
^@andrzejchm